### PR TITLE
[add] github actions

### PR DESCRIPTION
**Pull request by kazukazuinaina**
I added GitHub actions in this branch.
I have written below the results of GHA running on the fork destination.

## This PR's purpose

I thought it would be very convenient to be able to view the CI logs on github without logging into circleci.
When running on circleci, if a contributor wants to see the ci logs, they have to create a circleci account.
However, if we use github actions, we can eliminate the prerequisite of creating a circleci account. 

> https://github.com/kazukazuinaina/Dena.CodeAnalysis.Testing/pull/1/checks?check_run_id=3040689789

---

### Contribution License Agreement

- [x] By placing an "x" in the box, I hereby understand, accept and agree to be bound by the terms and conditions of the [Contribution License Agreement](https://dena.github.io/cla/).

### CODE REVIEW

**Comment by Kuniwak** [comment]
Ah, already CircleCI has been enabled, so adding more CI is unwelcome.

But I didn't know why CircleCI did not start for pull requests from forked repositories. "Build forked pull requests" on CircleCI project configuration seems enabled... I'm gonna ask to CircleCI support.

**Comment by Kuniwak** [comment]
I'm gonna keep this pull request opened until the CircleCI problem get solved.

**Comment by Kuniwak** [comment]
UPDATE: I have submitted a ticket for this problem. And I found a webhook from github.com to circleci.com of pull request events got 400.

**Comment by Kuniwak** [comment]
@kazukazuinaina I got a reply about this problem from CircleCI. It said reaching limit of @kazukazuinaina 's GitHub Access Token caused the problem. Any idea how this happened?

**Comment by kazukazuinaina** [comment]
@Kuniwak 
I deleted circleci's config and updated PR's description.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core SDK
```
Comment (Kuniwak) [inline .github/workflows/ci.yml:14]: Since SDK version 5, .NET Core should be just .NET.

**Review by Kuniwak — CHANGES_REQUESTED** [review]
Thank you for the explanation. I understood why GitHub Actions is better than CircleCI at here.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kazukazuinaina) [reply .github/workflows/ci.yml]: I fixed this workflow's name.

**Review by Kuniwak — APPROVED** [review]
LGTM

**Comment by kazukazuinaina** [comment]
@Kuniwak 
Thanks for your review and merging!

**Pull request merged**